guides/third-party-drivers.rst

```diff
@@ -0,0 +1,11 @@
+Third party Drivers
+======================
+
+Thanks to the ``Behat\Mink\Driver\DriverInterface`` interface, anyone can create a driver.
+
+Drivers that have implemented the mentioned interface and respect the driver test suite:
+
+.. toctree::
+    :maxdepth: 1
+
+    /third-party-drivers/phantomjs.rst
```

Comment:
IMO, the third-party drivers could be a section at the end of the ``drivers`` page instead of a separate page containing only the list, which has a single element for now.
This could be changed in the future if we have tens of third-party drivers, but I doubt it will be the case.